docs/app-integration-development/09-User-OAuth-Token-Code-Grant.md

```diff
@@ -132,10 +132,10 @@ The body of the request should include the following parameters: `client_id`, `c
 ```bash
 curl -X POST https://identity.pagerduty.com/oauth/token \
   --header "Content-Type: application/x-www-form-urlencoded" \
-  -d "grant_type=refresh_token" \
-  -d "client_id={CLIENT_ID}" \
-  -d "client_secret={CLIENT_SECRET}" \
-  -d "refresh_token=pdus+_1XBPWQQ_f85dd2f1-b906-478a-a9e6-678952529e4e"
+  --data-urlencode "grant_type=refresh_token" \
```

Comment:
This wasn't _quite_ correct before. Since we are sending `Content-Type: application/x-www-form-urlencoded` then the form field values themselves need to be url encoded.